policy-test/src/lib.rs

```diff
@@ -288,6 +318,58 @@ pub async fn logs(client: &kube::Client, ns: &str, pod: &str, container: &str) {
     }
 }
 
+impl Resource {
+    pub fn group(&self) -> String {
+        match self {
+            Self::EgressNetwork(_) => "policy.linkerd.io".to_string(),
+            Self::Service(_) => "core".to_string(),
+        }
+    }
+
+    pub fn kind(&self) -> String {
+        match self {
+            Self::EgressNetwork(_) => "EgressNetwork".to_string(),
+            Self::Service(_) => "Service".to_string(),
+        }
+    }
+
+    pub fn name(&self) -> String {
+        match self {
+            Self::EgressNetwork(e) => e.name_unchecked(),
+            Self::Service(s) => s.name_unchecked(),
+        }
+    }
+
+    pub fn namespace(&self) -> String {
+        match self {
+            Self::EgressNetwork(e) => e.namespace().unwrap(),
+            Self::Service(s) => s.namespace().unwrap(),
+        }
+    }
+
+    pub fn ip(&self) -> String {
+        match self {
+            Self::EgressNetwork(_) => "1.1.1.1".to_string(),
```

Comment:
Should this return an Option so that EgressNetwork can return a None instead of a dummy value?

Reply:
oh, this just needs to be any IP inside the EgressNework's cidr?

Reply:
Yes, that is it. Quite hacky but it works for now. If we bump into problems we can always make it more sophisticated.

Reply:
Would be good to have a comment explaining this, I think.